integration/test_collection_filter.py

```diff
@@ -695,31 +633,13 @@ def test_time_update_and_creation_time(collection_factory: CollectionFactory) ->
     assert objects_update[0].uuid == obj2_uuid
 
 
-def test_warning_old_filter(collection_factory: CollectionFactory) -> None:
-    collection = collection_factory(
-        properties=[Property(name="Name", data_type=DataType.TEXT)],
-        vectorizer_config=Configure.Vectorizer.none(),
-    )
-
-    uuids = [
-        collection.data.insert({"name": "Banana"}),
-        collection.data.insert({"name": "Apple"}),
-    ]
-    with pytest.warns(DeprecationWarning):
-        objects = collection.query.fetch_objects(filters=Filter("name").equal("Banana")).objects
-    assert len(objects) == 1
-    assert objects[0].uuid == uuids[0]
-
-
 def test_ref_count_filter(collection_factory: CollectionFactory) -> None:
     collection = collection_factory()
     collection.config.add_reference(
         ReferenceProperty(name="ref", target_collection=collection.name)
     )
 
-    if not collection._connection._weaviate_version.is_at_least(
-        1, 23, patch=3
-    ):  # todo change to patch=4 when it lands
+    if not collection._connection._weaviate_version.is_at_least(1, 23, patch=3):
```

Comment:
we can remove this completely? Same for the others

Reply:
Ah yes the tests too, missed those!